pkgpanda/build/__init__.py

```diff
@@ -1203,18 +1239,23 @@ def clean():
         # TODO(cmaloney): Run a wrapper which sources
         # /opt/mesosphere/environment then runs a build. Also should fix
         # ownership of /opt/mesosphere/packages/{pkg_id} post build.
-        cmd.run("package-builder", [
-            "/bin/bash",
-            "-o", "nounset",
-            "-o", "pipefail",
-            "-o", "errexit",
-            "/pkg/build"])
+        if is_windows:
+            cmd.run("package-builder", ["powershell.exe",
+                                        "-file",
+                                        PKG_DIR + "/build/windows.build.ps1"])
```

Comment:
The build script filename is hardcoded here, which doesn't look right.

Reply:
good catch!